.jenkins/pytorch/build.sh

```diff
@@ -30,10 +30,16 @@ cmake --version
 pip install -r requirements.txt || true
 
 if [[ "$BUILD_ENVIRONMENT" == *rocm* ]]; then
+  # This is necessary in order to cross compile (or else we'll have missing GPU device).
```

Comment:
one can target multiple GPU targets. we should consider doing so at least for fiji (gfx803).
see: https://github.com/RadeonOpenCompute/hcc#use-hcc_amdgpu_target-env-var

Reply:
@iotamudelta The only issue is increasing build times & memory usage.